modules/zip/src/parse-zip/local-file-header.ts

```diff
@@ -15,9 +15,12 @@ export type ZipLocalFileHeader = {
   fileDataOffset: bigint;
   /** Compressed size */
   compressedSize: bigint;
+  /** Compression method */
+  compressionMethod: number;
 };
 
 const offsets = {
```

Comment:
I think the file may minimize more effectively if you don't wrap these in an object
```suggestion

```

Reply:
Maybe add some comments about where these numbers come from, maybe some link to a spec.